esp/services/ws_access/ws_accessService.cpp

```diff
@@ -583,6 +583,7 @@ bool Cws_accessEx::onUserEdit(IEspContext &context, IEspUserEditRequest &req, IE
             throw MakeStringException(ECLWATCH_INVALID_SEC_MANAGER, MSG_SEC_MANAGER_IS_NULL);
         CLdapSecManager* ldapsecmgr = (CLdapSecManager*)secmgr;
         resp.setUsername(req.getUsername());
+        resp.setIsLDAPAdmin(ldapsecmgr->isSuperUser(context.queryUser()));
```

Comment:
You need to get the version of ESP client and set the IsLDAPAdmin if the version >= 1.13.
double version = context.getClientVersion();